code/modules/admin/antag_panel.dm

```diff
@@ -99,8 +99,8 @@ GLOBAL_VAR(antag_prototypes)
 
 	var/out = "<B>[name]</B>[(current && (current.real_name!=name))?" (as [current.real_name])":""]<br>"
 	out += "Mind currently owned by key: [key] [active?"(synced)":"(not synced)"]<br>"
-	out += "Assigned role: [assigned_role]. <a href='?src=[REF(src)];role_edit=1'>Edit</a><br>"
-	out += "Faction and special role: <b><font color='red'>[special_role]</font></b><br>"
+	out += "Station role: [get_station_role(TRUE)] <br>" //  <a href='?src=[REF(src)];role_edit=1'>Edit</a><br> // Evildragon: commenting the role edit. it needs another refactor later. excessive job refactor was a pain already
```

Comment:
The antag panel needs a redesign anyway, it's incredibly dated.

Ideally we should have an entire admin "role editor" that lets you control antag datums, mind roles, jobs, etc and allow you to apply this to players all at once or spawn players with this